src/scripts/safari/SafariPokemonList.ts

```diff
@@ -56,6 +58,69 @@ class SafariPokemonList {
+            new SafariEncounter('Ledyba', 3),
+            new SafariEncounter('Spinarak', 3),
+            new SafariEncounter('Paras', 3),
+            new SafariEncounter('Scyther', 3),
+            new SafariEncounter('Pinsir', 2),
+            new SafariEncounter('Pineco', 3),
+            new SafariEncounter('Shuckle', 2),
+            new SafariEncounter('Wurmple', 5, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Silcoon', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Beautifly', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Cascoon', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Dustox', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Masquerain', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Nincada', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Kricketot', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Kricketune', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Combee', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Vespiquen', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Yanmega', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Sewaddle', 5, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Swadloon', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Leavanny', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Venipede', 5, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Whirlipede', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Scolipede', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Dwebble', 5, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Durant', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Scatterbug', 5, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Spewpa', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Grubbin', 5, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Charjabug', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Vikavolt', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Cutiefly', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Blipbug', 5, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Dottler', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Orbeetle', 1, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Sizzlipede', 2, [SafariEnvironments.Grass], true),
+            new SafariEncounter('Snom', 2, [SafariEnvironments.Grass], true),
+
+            // Water
+            new SafariEncounter('Krabby', 1), //TODO Replace if possible
```

Comment:
Could surskit work, since it’s bug/water from johto? Or is there a reason to keep it out of the contest that I’m not thinking about? (If the issue is type eggs, fair enough, my next best theory is yanma since it’s literally based on a dragonfly, which are usually found in marshes)

Reply:
Surskit is from Hoenn

Reply:
Whoops. I always think surskit is johto and slugma is hoenn. My bad. Sticking with my yanma backup suggestion then.